Formula/openalpr.rb

```diff
@@ -16,23 +16,58 @@ class Openalpr < Formula
   depends_on "leptonica"
   depends_on "libtiff"
   depends_on "log4cplus"
-  depends_on "opencv@2"
+  depends_on "opencv"
   depends_on "tesseract"
 
+  uses_from_macos "curl"
+
+  # A photo of licence plate from https://commons.wikimedia.org/wiki/File:California_2018_license_plate_(USA).jpg
+  resource "testdata" do
+    url "https://upload.wikimedia.org/wikipedia/commons/b/b0/California_2018_license_plate_%28USA%29.jpg"
+    sha256 "dd58b62f4f75690280dd0b021b19422cef7f4f4381664d3ee512106e6f491e5c"
+  end
+
+  # Make compatible with opencv 4
+  # See https://github.com/openalpr/openalpr/pull/878
+  # Remove in the next release
+  patch do
+    url "https://github.com/openalpr/openalpr/commit/a35f0d688e546392bf15e0d1d4ef73b6bc8d179d.patch?full_index=1"
+    sha256 "c22c0773182badc6ff7a65c0a423b7a01d722788be192f1b4245b1bad8d15089"
+  end
+
+  # Fix compatibility with tesseract 4
+  # See https://github.com/openalpr/openalpr/pull/693
+  # Remove in the next release
+  patch do
+    url "https://github.com/openalpr/openalpr/commit/28e0b0fa95c12923138768d670b852242a750d0c.patch?full_index=1"
+    sha256 "3c99b8fe00f7fbdc3840cdf0453154fd8d88bba90de7636137697194b9632b30"
+  end
+
   def install
+    # Help cmake to find tesseract
+    # See https://github.com/Homebrew/homebrew-core/pull/85797#issuecomment-925985674
+    inreplace "src/cmake_modules/FindTesseract.cmake" do |s|
+      s.gsub!(%r{^\s*HINTS "/usr/include"$}, "HINTS \"#{Formula["tesseract"].opt_include}/tesseract\"")
+      s.gsub!(%r{^\s*HINTS "/usr/lib"$}, "HINTS \"#{Formula["tesseract"].opt_lib}/tesseract\"")
+    end
```

Comment:
Does setting `Tesseract_PKGCONF_INCLUDE_DIRS` and `Tesseract_PKGCONF_LIBRARY_DIRS` instead of doing an `inreplace` not work?

Reply:
Yep, it works, thanks!

Reply:
Your `inreplace` is still here...

Reply:
It turns out you need to commit the changes before pushing it! What a surprise 😄